consensus/hotstuff/pacemaker.go

```diff
@@ -40,29 +40,28 @@ type PaceMaker interface {
 	// CurView returns the current view.
```

Comment:
We also have `HighestQC` in the Implementation Draft. Should that be here, or should we retrieve that directly from the `Persister` component?

Reply:
We should have it here, missed it.